modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules.module

```diff
@@ -63,12 +63,9 @@ function facebook_instant_articles_api_remove_action($node, $context = array())
  */
 function fb_instant_articles_api_rules_import_article($node) {
   if (isset($node->nid)) {
-    $layout_settings = fb_instant_articles_display_get_node_layout_settings($node->type);
-    $wrapper = \Drupal\fb_instant_articles_display\DrupalInstantArticleDisplay::create($node, $layout_settings);
```

Comment:
Were these lines simply redundant because they also happen in [`fb_instant_articles_display_node_load`](https://github.com/BurdaMagazinOrg/module-fb_instant_articles/blob/7.x-1.x/modules/fb_instant_articles_display/fb_instant_articles_display.module#L188) (just trying to understand)?